pkg/sql/schema_changer_test.go

```diff
@@ -1476,6 +1476,9 @@ CREATE TABLE t.test (k INT PRIMARY KEY, v INT);
 
```

Comment:
Because `rows` escapes and is used outside of the `SucceedsSoon` loop. Switched to a `Fatal`.